collector/netdev_common.go

```diff
@@ -26,12 +27,14 @@ import (
 )
 
 var (
-	netdevIgnoredDevices = kingpin.Flag("collector.netdev.ignored-devices", "Regexp of net devices to ignore for netdev collector.").Default("^$").String()
+	netdevIgnoredDevices = kingpin.Flag("collector.netdev.ignored-devices", "Regexp of net devices to ignore for netdev collector (mutually exclusive to accept-devices).").String()
```

Comment:
I'm wondering if we should make this match the `systemd` collector flag names.

Ie: `collector.netdev.device-whitelist` and `collector.netdev.device-blacklist`